mathics/builtin/arithfns/basic.py

```diff
@@ -240,8 +240,7 @@ class Minus(PrefixOperator):
 
     def apply_int(self, x, evaluation):
         "Minus[x_Integer]"
-
-        return Integer(-x.to_sympy())
```

Comment:
Here is an example of the looseness of indicating what type we have may have contributed to the sloppiness.